packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.tsx

```diff
@@ -201,18 +226,36 @@ function LoadedFilterDialogContent(props: LoadedFilterDialogContentProps) {
       <div className="presentation-instance-filter-dialog-bottom-container">
         <div>{filterResultsCountRenderer ? <ResultsRenderer buildFilter={getFilterInfo} renderer={filterResultsCountRenderer} /> : null}</div>
         <Dialog.ButtonBar className="presentation-instance-filter-button-bar">
-          <Button className="presentation-instance-filter-dialog-apply-button" styleType="high-visibility" onClick={applyButtonHandle} disabled={isDisabled}>
-            {translate("instance-filter-builder.apply")}
-          </Button>
-          <Button className="presentation-instance-filter-dialog-close-button" onClick={onClose}>
-            {translate("instance-filter-builder.cancel")}
-          </Button>
+          {toolbarButtonsRenderer ? (
+            toolbarButtonsRenderer({ handleApply, handleReset, handleClose: onClose })
+          ) : (
+            <ToolbarButtonsRenderer handleApply={handleApply} handleReset={handleReset} handleClose={onClose}></ToolbarButtonsRenderer>
+          )}
         </Dialog.ButtonBar>
       </div>
     </>
   );
 }
 
+function ToolbarButtonsRenderer({ handleApply, handleClose, handleReset }: FilteringDialogToolbarHandlers) {
+  return (
+    <Dialog.ButtonBar className="presentation-instance-filter-button-bar">
```

Comment:
`Dialog.ButtonBar` can be removed it is added in `LoadedFilterDialogContent` component.